sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/AbstractDistributeTest.java

```diff
@@ -177,6 +199,8 @@ public Void answer(InvocationOnMock invocation) {
         mockStatic(NetworkStateHelper.class);
         mNetworkStateHelper = mock(NetworkStateHelper.class, new Returns(true));
         when(NetworkStateHelper.getSharedInstance(any(Context.class))).thenReturn(mNetworkStateHelper);
+        PowerMockito.spy(HttpUtils.class);
```

Comment:
Usually spy takes an instance. Is it working?

Reply:
The target here is mock static method